pages/common/gatsby.md

```diff
@@ -0,0 +1,23 @@
+# gatsby
+
+> Static site generator for React.
+
+- Create a new site:
+
+`gatsby new {{site_name}}`
+
+- Create a new site with a Gatsby 'starter':
+
+`gatsby new {{site_name}} {{url_of_starter_github_repo}}`
+
+- Start a live-reloading developer environment:
```

Comment:
Perhaps `development server` might be aid clarity here? Unless it does more than starts a server?